spec/jobs/dashboard/applicant_email_job_spec.rb

```diff
@@ -26,13 +26,42 @@ module Dashboard
           end
         end
 
-        context 'job is not in the suspended list' do
+        context 'job is in the suspended list' do
           before { allow(HostEnv).to receive(:environment).and_return(:development) }
           it 'does not call the applicant email job' do
             expect_any_instance_of(Dashboard::SingleObject::ApplicantEmail).not_to receive(:run)
             subject
           end
         end
+
+        context 'job is sending email to an Apply team email' do
+          let(:applicant) { create :applicant, email: Rails.configuration.x.applyemail.suffix }
+          let(:application) { create :legal_aid_application, applicant: applicant }
+
+          context 'in production environment' do
+            before do
+              allow(HostEnv).to receive(:environment).and_return(:production)
+              allow(Rails.env).to receive(:production?).and_return(true)
+            end
+
+            it 'does not call the applicant email job' do
+              expect_any_instance_of(Dashboard::SingleObject::ApplicantEmail).not_to receive(:run)
+              subject
+            end
+          end
+
+          context 'not in production environment' do
+            before do
+              allow(HostEnv).to receive(:environment).and_return(:production)
```

Comment:
If we aren't changing the `HostEnv` value,  this could be in a 
```ruby 
before { allow(HostEnv).to receive(:environment).and_return(:production) } 
```
on line 40?